app/services/personal_key_formatter.rb

```diff
@@ -12,4 +12,9 @@ def self.regexp
   def self.regexp_string
     REGEXP
   end
+
+  def self.code_length
+    # TODO: Put this somewhere else!!
```

Comment:
It's a combination of `RandomPhrase::WORD_LENGTH` and `Figaro.env.recovery_code_length.to_i` so I think...

```
num_words = Figaro.env.recovery_code_length.to_i
(RandomPhrase::WORD_LENGTH * num_words) + (num_words - 1)
```